scripts/gui/microbe_editor.mjs

```diff
@@ -354,6 +354,7 @@ function onResumeClickedEditor(){
 
 function onExitToMenuClickedEditor(){
     if(common.isInEngine()){
+        document.getElementById("pauseOverlayEditor").style.display = "none";
```

Comment:
Why is this needed? Is this related to the freebuild editor or a general fix?

Reply:
Oh this is just a general fix.